### PR TITLE
Edit Photos

### PR DESCRIPTION
**Pull request by JancoDVTDev**
## Photo Edit Feature
User can now edit photos in their album and save them to their camera roll.
Feature mostly iOS 13 drivin.

### Adjust Photo
- User can choose 5 adjustment options.
- Available adjustments: Vibrance - Exposure - Gamma - Vignette - Tint.

### Filter Photo
- There is 5 predefined filters. 
- Available Filters: CamShare - Cold - Vivid Cold - Warm - Vivid Warm.
- CamShare is an App Custom filter.

### Saving Image
- Exporting the image will save it to the user’s camera roll.
- Does not effect the photo in the app; Feature to be added.

### Images

IMG_3271

IMG_3273

IMG_3274

IMG_3278

### CODE REVIEW

File: camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 1]
```diff
@@ -8,7 +8,7 @@
          BreakpointExtensionID = "Xcode.Breakpoint.FileBreakpoint">
```
Comment (bgouws) [inline camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:8]: Avoid uploading xcuserdata.

**Review by AkuaQ — APPROVED** [review]

**Review by bgouws — APPROVED** [review]

File: camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist (hunk 1 above)
Comment (JancoDVTDev) [reply camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:8]: Do you know where to put the custom icons then ?

Comment (Zamo22) [reply camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:8]: This file is purely for Breakpoints though, we should avoid adding this as Brandon mentioned

File: camshare/camshare/AppDelegate.swift [hunk 2]
```diff
@@ -11,6 +11,8 @@ UIImage(named: "Image4"), UIImage(named: "Image5"), UIImage(named: "Image6"),
 UIImage(named: "Image7"), UIImage(named: "Image8"), UIImage(named: "Image9"),
 UIImage(named: "Image10"), UIImage(named: "Image11"), UIImage(named: "Image12")]
 
+var imageToBeEdited = UIImage()
```
Comment (Zamo22) [inline camshare/camshare/AppDelegate.swift:14]: Why is this in AppDelegate? If it has to be that is fine but is there no way we could have it somewhere else ?

**Comment by Zamo22** [comment]
Feature looks pretty cool 👍

File: camshare/camshare/AppDelegate.swift (hunk 2 above)
Comment (JancoDVTDev) [reply camshare/camshare/AppDelegate.swift:14]: Actually not using that variable, will remove it.

File: camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist (hunk 1 above)
Comment (JancoDVTDev) [reply camshare/camshare.xcworkspace/xcuserdata/jerasmus.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist:8]: Cool thanks Zaheer, Brandon explained on slack

**Pull request merged**